neo/Helper.cs

```diff
@@ -307,5 +308,21 @@ internal static long WeightedAverage<T>(this IEnumerable<T> source, Func<T, long
                 yield return resultSelector(item, weight);
             }
         }
+
+
+        /// <summary>
+        /// load configuration with different Environment Variable
+        /// </summary>
+        /// <param name="config"></param>
+        /// <returns></returns>
+        public static IConfigurationRoot LoadConfig(string config)
```

Comment:
I still think this can't be reused. Because the config files are not always at the root application directory.

Reply:
As my test, we can pass the parameter "config" like "Plugins/config" which can surpport different directory.

Reply:
Most of the plugin has only one config file. In this case, if they use this method, they will try to read from a file that does not exist.

Reply:
You are right . As far as I know, the plugin config files rarely change along with the network, so these scenarios wouldn't use this method. The most common usage is loading "config.json" and "protocal.json".  Mark it public so neo-cli can call it to load "config.json".